app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -1,9 +1,10 @@
 module ProviderInterface
   class DecisionsController < ProviderInterfaceController
     before_action :set_application_choice
-    before_action :requires_provider_user_make_decisions_permission
     before_action :requires_provider_change_response_feature_flag, only: %i[new_withdraw_offer confirm_withdraw_offer withdraw_offer]
 
+    include RequiresMakeDecisionsPermission
```

Comment:
:+1: Pulling this out makes a lot of sense to me. Would it be more Rails-y to mix in the module in `ProviderInterfaceController` without the `included...` block then continue to use a traditional `before_action : requires_provider_user_make_decisions_permission`?

Reply:
Ok, I'll remove the `included...` block and use explicit before actions.